internal/pkg/template/service.go

```diff
@@ -93,8 +93,10 @@ type ServiceOpts struct {
 	Autoscaling *AutoscalingOpts
 
 	// Additional options that're not shared across all service templates.
-	HealthCheck        *ecs.HealthCheck
-	RulePriorityLambda string
+	HealthCheck                *ecs.HealthCheck
+	RulePriorityLambda         string
+	DesiredCountLambda         string
+	DesiredCountLambdaUpdateID string
```

Comment:
What do think of this idea?

Let's not require `DesiredCountLambdaUpdateID` as a field and instead let's move the `randomUUID` function to this package and add it to:
https://github.com/aws/copilot-cli/blob/3ce45ffaf811b3818dc7e2a0deccc7a1196c7446/internal/pkg/template/service.go#L134-L143

so this way in the CFN template itself we can write:
```yaml
UpdateID: {{ .randomUUID }}
```
and if this call errors then the execution will stop:
>  If it has two and the returned error is non-nil, execution terminates
  and an error is returned to the caller as the value of Execute.
https://golang.org/pkg/text/template/

Reply:
I think that will remove confusion on what is the purpose of `DesiredCountLambdaUpdateID`

Reply:
good idea! that's neat